src/extension-client/remix-extension.ts

```diff
@@ -15,7 +16,6 @@ export class RemixExtension<T extends Api = any> {
   private handshake: () => any
   protected currentRequest: PluginRequest
   public isLoaded = false
-  public devMode = false
```

Comment:
could we keep `devMode`? as we might need to use it for something else than filtering out the right dev port.

Reply:
Then we can just check is `devModePort` exists ?
Maybe we can have a default `8080` value in the `setDevMode` function in case someone forget to fill the argument : 
```javascript
public setDevMode(devModePort: number | string = 8080) {
  this.devModePort = devModePort
}
```

Reply:
> Then we can just check is devModePort exists ?

`devMode` support could at some point involve other property than `devModePort`. 
That's why a global switch `devMode` could be handy

Reply:
We can have an object `devMode` with a property `port` if you think it's better. But we should be careful not to have too much one-property-object that linger in the code.